config.py

```diff
@@ -893,6 +893,24 @@
 
         hidden_category = 'Kategori_tersembunyi',
     ),
+
+    eu = dict(
+        lang_name = 'Euskara',
+        lang_dir = 'ltr',
+        database = 'euwiki_p',
+        wikipedia_domain = 'eu.wikipedia.org',
+        citation_needed_category = 'Erreferentzia behar duten adierazpenak dauzkaten artikuluak',
```

Comment:
```suggestion
        citation_needed_category = 'Erreferentzia_behar_duten_adierazpenak_dauzkaten_artikuluak',
```

This needs to have underscores instead of spaces, please.

Reply:
Done!